### PR TITLE
SRCH-4873 Data Migration for Launch Support - all sites with Custom theme - Visual design db column settings

### PR DESCRIPTION
**Pull request by krbhavith**
## Summary
- Migrate fonts and colors from old SERP to Visual Design settings when Visual Design settings are not modified from default setting.
 
### Checklist
Please ensure you have addressed all concerns below before marking a PR "ready for review" or before requesting a re-review. If you cannot complete an item below, replace the checkbox with the ⚠️ `:warning:` emoji and explain why the step was not completed.
 
#### Functionality Checks
 
- [x] You have merged the latest changes from the target branch (usually `main`) into your branch.
  
- [x] Your primary commit message is of the format **SRCH-#### \<description\>** matching the associated Jira ticket.

- [x] PR title is either of the format **SRCH-#### \<description\>** matching the associated Jira ticket (i.e. "SRCH-123 implement feature X"), or **Release - SRCH-####, SRCH-####, SRCH-####** matching the Jira ticket numbers in the release.
 
- [x] Automated checks pass. If Code Climate checks do not pass, explain reason for failures:
 
#### Process Checks

- [x] You have specified at least one "Reviewer".

### CODE REVIEW

File: app/models/custom_font_and_color_theme_updater.rb [hunk 1]
```diff
@@ -0,0 +1,70 @@
+# frozen_string_literal: true
+
+class CustomFontAndColorThemeUpdater
+  VISUAL_DESIGN_FONT = %w[primary_navigation_font_family header_links_font_family footer_and_results_font_family identifier_font_family].freeze
```
Comment (ddzz) [inline app/models/custom_font_and_color_theme_updater.rb:4]: This appears to be the same as `Affiliate::FONT_FIELDS`. We shouldn't duplicate any of these attributes in these constants.

File: app/models/custom_font_and_color_theme_updater.rb
[the remark this answers is not in the document]
Comment (krbhavith) [reply app/models/custom_font_and_color_theme_updater.rb:1]: @ddzz I agree, generally these methods will go in Affiliate class. But this is mostly one time task and we don't have to use those methods frequently. Also I followed the existing /Users/bhavith/workspace/search-gov/app/models/font_and_color_updater.rb.

[the remark this answers is not in the document]
Comment (ddzz) [reply app/models/custom_font_and_color_theme_updater.rb:1]: @krbhavith Is this a temporary file that can eventually be flagged for deletion? If so, we should create a task for that so we don't forget. And also for the other updater model you mentioned.

**Review by stevenbarragan — APPROVED** [review]
LGTM

**Pull request merged**